CMakeLists.txt

```diff
@@ -614,7 +622,7 @@ foreach(tar ${srtpack_libspec_common})
 endforeach()
 
 
-set (SRT_LIBS_PRIVATE ${SRT_LIBS_PRIVATE} ${PTHREAD_LIBRARY})
+set (SRT_LIBS_PRIVATE ${SRT_LIBS_PRIVATE} ${PTHREAD_LIBRARY} ${GNUSTL_LIBRARIES} ${GNUSTL_LDFLAGS})
```

Comment:
Please add this in line 288, separately. If this introduces a dependency problem, use a separate variable for a purpose of providing "standard C++" library.

Please note that this is a variable, which's value will be pasted into the `pkg-config` file. That's why it doesn't use `list(APPEND)`.